src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuide.spec.tsx

```diff
@@ -0,0 +1,59 @@
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { Store } from '../../../../types';
+import createTestStore from '../../../../../test/createTestStore';
+import createTestServices from '../../../../../test/createTestServices';
+import MessageProvider from '../../../../MessageProvider';
+import * as Services from '../../../../context/Services';
+import UsingThisGuideButton from './UsingThisGuideButton';
+import UsingThisGuideBanner from './UsingThisGuideBanner';
+
+describe('Using this guide', () => {
+  let store: Store;
+  let services: ReturnType<typeof createTestServices>;
```

Comment:
`<UsingThisGuideButton/>` and `<UsingThisGuideBanner/>` are not using `store` / `services`
It can be added later when we'll add analytics event